File: Addition.cpp
#include<iostream>
using namespace std;

class Summation
{ 
   public:
   int iNo1;
   int iNo2;
   
	Summation(int a,int b)
	{
		iNo1=a;
	    iNo2=b;
	}
	
	~Summation()
	{
		cout<<"inside destructor\n";
	}
int Addition()
{
	int ians=0;
	ians=iNo1+iNo2;
	return ians;
}
};
int main()
{
	 
	int iValue1=0,iValue2=0,iRet=0;
	
	cout<<"Enter first number"<<"\n";
	cin>>iValue1;
	
	cout<<"Enter second number"<<"\n";
	cin>>iValue2;
	
	Summation sobj(iValue1,iValue2);
	
	iRet=sobj.Addition();
	
	cout<<"Addition is:"<<iRet<<"\n";
	
	return 0;
}

File: Addition.java
import java.lang.*;

class Summation
{
	public int iNo1;
	public int iNo2;
	
	public Summation(int a,int b)
	{
		this.iNo1=a;
		this.iNo2=b;
	}
	 public int Addition()
	{
		int ans=iNo1+iNo2;
		return ans;
	}
}

class Addition
{
	public static void main(String arg[])
	{
		Summation sobj=new Summation(10,20);
		
		int iRet=sobj.Addition();
		System.out.println(iRet);
	}
}